### DISCUSSION TITLE
How to get gas price ( avg, high , low ) of base chain , I checked base scan api but they don't price this value

### DISCUSSION THREAD

**Discussion by SrishtiLodhi** [blockscout/blockscout#10256] [Q&A]
How to get gas price ( avg, high , low ) of base chain , I checked base scan api but they don't price this value

**Comment by sl1depengwyn** [comment]
Check `"gas_prices"` in https://base.blockscout.com/api/v2/stats

**Comment by SrishtiLodhi** [comment]
Does base provide it from their end?

**Reply by sl1depengwyn** [reply]
It's calculated on Blockscout side
